.github/workflows/perf-budgets.yml

```
@@ -0,0 +1,80 @@
+name: Performance Budgets
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)